src/components/Popover/index.test.tsx

```diff
@@ -14,7 +14,7 @@
   limitations under the License.                                                                              *
  ******************************************************************************************************************** */
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, fireEvent, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
```

Comment:
are we using `waitForElementToBeRemoved` ?

Reply:
Ooh not any more, will delete!